e2e/features/build-case.feature

```diff
@@ -10,12 +10,16 @@ Feature: Build case
     And I upload respondent evidence
     And I switch to be a `Legal Rep`
 
-  @create-direction @RIA-480 @RIA-600
+  @create-direction @RIA-480 @RIA-600 @RIA-927
   Scenario: Upload Case argument and evidence
 
     When I select the `Build your case` Next step
     Then I am on the `Build your case` page
+    And the `Continue` button is disabled
+    And I should see the text `You must upload your appeal skeleton argument to submit your case`
+
     When I upload `{@CaseArgument.pdf}` for the `Appeal skeleton argument` field
+    Then the `Continue` button is enabled
     And I type `This is the case argument` for the `Describe the document (Optional)` field
```

Comment:
I would change it to When here, to maintain test flow

Reply:
Changes applied